.github/workflows/docs.yml

```diff
@@ -47,18 +47,21 @@ jobs:
           packages: libosmesa6-dev libgl1-mesa-dev python3-tk pandoc git-restore-mtime
           version: 3.0
 
-      - name: Install custom OSMesa VTK variant
-        run: pip install https://github.com/pyvista/pyvista-wheels/raw/main/vtk_osmesa-9.2.5-cp39-cp39-linux_x86_64.whl
-
       - name: Install PyVista and dependencies
         run: |
           pip install -e . --no-deps
           pip install -r requirements_docs.txt
 
+      - name: Install custom OSMesa VTK variant
+        run: |
+          pip uninstall vtk -y
+          pip install https://github.com/pyvista/pyvista-wheels/raw/main/vtk_osmesa-9.2.5-cp311-cp311-linux_x86_64.whl
```

Comment:
@akaszynski, @MatthewFlamm, @AlejandroFernandezLuces: this is the issue I am referring to in https://github.com/pyvista/pytest-pyvista/pull/41

Reply:
I think there might be other issues.  For example here is the relevant lines from the ongoing doc build.  Note that vtk isn't installed by `pip install -e . --no-deps` as expected.  the package from pyvista-wheels is `vtk-osmesa`.

```txt
Run pip uninstall vtk -y
WARNING: Skipping vtk as it is not installed.
Collecting vtk-osmesa==9.2.5
  Downloading https://github.com/pyvista/pyvista-wheels/raw/main/vtk_osmesa-9.2.5-cp311-cp311-linux_x86_64.whl (76.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 76.1/76.1 MB 38.6 MB/s eta 0:00:00
Installing collected packages: vtk-osmesa
Successfully installed vtk-osmesa-9.2.5
```

Reply:
Sorry. I commented on the wrong spot. `pytest-pyvista` isn't used here 🤦🏻‍♂️

Reply:
See https://github.com/pyvista/pyvista/pull/3704/files#r1094927082